### PR TITLE
Improve docs about running nextron in development

### PR DESCRIPTION
**Pull request by andirsun**
closes https://github.com/saltyshiomix/nextron/issues/337

### CODE REVIEW

**Review by saltyshiomix — COMMENTED** [review]
Could you fix typo in ordered lists?
（1, 2, 4 => 1, 2, 3）

Thanks in advance,

スクリーンショット 2023-04-06 15 27 54

**Comment by andirsun** [comment]
> Could you fix typo in ordered lists? （1, 2, 4 => 1, 2, 3）
> 
> Thanks in advance,
> 
> スクリーンショット 2023-04-06 15 27 54

@saltyshiomix Fixed! is ready to merge.

**Comment by saltyshiomix** [comment]
@andirsun 

Sorry for late reply. And big thanks for your efforts!

LGTM 🎉

File: README.md [hunk 1]
```diff
@@ -402,6 +402,23 @@ $ pnpm dev # default is examples/basic-javascript
 $ pnpm dev <EXAMPLE-FOLDER-NAME>
 ```
 
+### Developing for your own project
```
Comment (saltyshiomix) [inline README.md:405]: 🎉

File: README.md [hunk 2]
```diff
@@ -402,6 +402,23 @@ $ pnpm dev # default is examples/basic-javascript
...
+```
+$ cd nextron
+$ npm install
+$ npm run build
+$ npm link 
+```
+2. Install linked nextron in your project
+```
+$ cd your-project
+$ npm install -D @babel/runtime-corejs3 # required for nextron
```
Comment (saltyshiomix) [inline README.md:417]: [MEMO] I'll add `@babel/runtime-corejs3` to nextron as a depenency 🙇

**Review by saltyshiomix — APPROVED** [review]
LGTM <3

**Pull request merged**